config/default.yml

```diff
@@ -247,6 +247,11 @@ PreCommit:
     description: 'Check for file execute permissions'
     quiet: true
 
+  Fasterer:
+    enabled: false
+    description: 'Suggest some speed improvements'
+    required_executable: 'fasterer'
```

Comment:
Seems like this should have `include: '**/*.rb'` since it's Ruby-specific and doesn't require a full repo context like Brakeman does.